packages/components/psammead-radio-schedule/src/StartTime/__snapshots__/index.test.jsx.snap

```diff
@@ -142,6 +142,10 @@ exports[`StartTime should render LTR correctly 1`] = `
     >
       14:54
     </time>
+    <span
```

Comment:
Not sure about adding an empty span just for a line, can't we live without this in IE??

Reply:
I'd rather we kept using the pseudo-element, and just didn't show the line in IE. We can run it past UX?

Reply:
Will reach out to UX and revert

Reply:
Fixing this issue on IE has been deemed as low priority for now, and as something that should be added to the tech debt backlog. I will revert the changes made to remove the pseudo styling and only include the fix for Safari in this PR.  

**Note: Adding additional mark up to fix this is not an appropriate solution as mentioned by** @greenc05 [above](https://github.com/bbc/psammead/pull/3318#discussion_r401015655)